var/spack/repos/builtin/packages/py-plotly-helper/package.py

```diff
@@ -15,11 +15,13 @@ class PyPlotlyHelper(PythonPackage):
     url = "https://pypi.io/packages/source/p/plotly-helper/plotly-helper-0.0.7.tar.gz"
 
     version('develop', branch='master')
+    version('0.0.8', sha256='fda9f3c744c679b74ef2f34fa0e0fd00426a95b6e8604a57904c7438628f9233')
```

Comment:
`py-plotly-helper` is not required by other packages and it's not built in any version.
Probably we can keep it because it's an optional dependency for some pip packages, but at the moment it seems useless.

Reply:
Thank you for checking that. I suggest we delete it then.

Reply:
It's ok for me.

Reply:
I just checked. The replacement for `py-morphology-repair-workflow` contains a direct dependency to plotly-helpers (no extras). I will keep it.